client/request.go

```
@@ -617,8 +728,18 @@
 	*fasthttp.Args
 }
 
+// Keys method returns all keys in the query params.
+func (f *QueryParam) Keys() []string {
+	keys := make([]string, f.Len())
+	f.VisitAll(func(key, value []byte) {
+		keys = append(keys, utils.UnsafeString(key))
```

golangci-lint (1.62.2)

append to slice `keys` with non-zero initialized length

(makezero)